_sass/blocks/_revenues_subpage_steps.scss

```diff
@@ -28,16 +28,27 @@ $onshore-renewables-secondary-color: $blue;
   @include respond-to(medium-up) {
     display: block;
     margin-top: 40%;
-    @include span-columns(3);
+    @include span-columns(2);
+    &[accordion-open=true] {
+      @include span-columns(3);
+    }
   }
+
 }
 
+
+.revenues_subpage-dyk_heading,
+.revenues_subpage-dyk_text,
+.revenues_subpage-dyk_text p {
+  @include heading('para-sm');
+}
 .revenues_subpage-dyk_heading {
+  font-weight: $weight-bold;
```

Comment:
Properties should be ordered background-color, color, font-weight, margin, padding, text-align